src/SFA.DAS.RoatpGateway.Web/Attributes/ExternalApiExceptionFilter.cs

```diff
@@ -11,7 +11,7 @@ public override void OnException(ExceptionContext filterContext)
             if (filterContext.Exception is ExternalApiException)
             {
                 filterContext.ExceptionHandled = true;
-                filterContext.Result = new RedirectToActionResult("ExternalApisUnavailable", "RoatpShutterPage", new { });
+                filterContext.Result = new RedirectToActionResult("ServiceUnavailable", "RoatpShutterPage", new { });
```

Comment:
This is a very specific filter which has a specific name for purpose, so why are we changing this?

Reply:
The error is really always going to go to a service unavailable page, so I thought it might be better to have the url drive to ..../ServiceUnavailable rather than .../ExternalApisUnavailable.     I don't feel very strongly about it though.  Shall I just change it back?